Assets/Scripts/BranchScript.cs

```diff
@@ -17,29 +18,19 @@ private void Start()
         _spawner = GameObject.Find("Branch Spawner").GetComponent<BranchSpawner>();
         _transform = GetComponentInChildren<Transform>();
         _transform.localScale = new Vector2(xScale, 0.75f);
+        _boxColliderXScale = GetComponent<BoxCollider2D>().offset.x;
 
-        if (treeLocation == BranchSpawner.Direction.Left) 
+        if (treeLocation == BranchSpawner.Direction.Right) 
         {
-            _transform.Translate(new Vector2(xScale/2, 0f));
-        } 
-        else 
-        {
-            _transform.Translate(new Vector2(-xScale/2, 0f));
+            GetComponent<SpriteRenderer>().flipX = true;
+            GetComponent<BoxCollider2D>().offset = new Vector2(-_boxColliderXScale, GetComponent<BoxCollider2D>().offset.y);
```

Comment:
If we have a private member called _collider instead of _boxColliderXscale, this can be simplified:

_collider.offset = new Vector2(-_collider.offset.x, _collider.offset.y);